blocks/library/gallery/index.js

```diff
@@ -166,19 +166,12 @@ export const settings = {
 		],
 	},
 
-	getEditWrapperProps( attributes ) {
-		const { align } = attributes;
-		if ( 'left' === align || 'right' === align || 'wide' === align || 'full' === align ) {
-			return { 'data-align': align };
-		}
-	},
-
 	edit: GalleryBlock,
 
 	save( { attributes } ) {
-		const { images, columns = defaultColumnsNumber( attributes ), align, imageCrop, linkTo } = attributes;
+		const { images, columns = defaultColumnsNumber( attributes ), imageCrop, linkTo } = attributes;
 		return (
-			<ul className={ `align${ align } columns-${ columns } ${ imageCrop ? 'is-cropped' : '' }` } >
+			<ul className={ `columns-${ columns } ${ imageCrop ? 'is-cropped' : '' }` } >
```

Comment:
Unrelated to your changes, but this is begging for `classnames` 😄

Reply:
Let's add in this PR this change, too 👍